CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Change log
 
+## 5.0.0 (2018-06-18)
+
+Breaking changes: 
```

Comment:
Another 'breaking' change: Always include documentation in JSON and YAML output.

Reply:
Updated.